src/components/HappySlider/HappySlider.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'
+
+import './HappySlider.scss'
+
+// Relative path does not work for css Variable
+// import images into a variable
+import smiley1 from './Smileys/smiley1.svg'
+import smiley2 from './Smileys/smiley2.svg'
+import smiley3 from './Smileys/smiley3.svg'
+import smiley4 from './Smileys/smiley4.svg'
+import smiley5 from './Smileys/smiley5.svg'
```

Comment:
I can try. Any suggestions on how before I begin?

Reply:
This looks promising:
https://stackoverflow.com/questions/42118296/dynamically-import-images-from-a-directory-using-webpack

Although, It seems to require more code than what we currently have! Lol.

Reply:
Or maybe this? It entails creating a separate component.

https://stackoverflow.com/questions/48560592/is-it-possible-to-import-a-group-of-images-as-an-array-create-react-app-projec

Reply:
Personally I think the second one looks cleaner (and easier possibly?) let us know what you try out and if it works, I'm interested in this!

Reply:
Yeah, I like the second one too, but it may not be much of an upgrade for what we currently have. It would just import the files into a different component. I could see the value in it if we had hundreds of images, but we only have five images. @asooge what are your thoughts on this? Is this separate component what you were thinking of?

Reply:
Just curious, do we actually need to import the images into `HappySlider.js`? I see them getting declared as property values for `:root` in the scss file.   `style.setProperty(propertyName, value, priority)` only needs the first parameter. Second and third are all optional.

Reply:
The value of the slider is determined in `HappySlider.js`. So we need the second parameter to set the value of the background image. 

The `:root` in the scss file is setting the variables. We may not even need to have the images in the variable to begin with. I'll check that out.

Reply:
Oh I see👍🏼 I misunderstood what those root property values were doing.

Reply:
Can you create a new file and `export const smileys = [ ... ]`

then `import { smileys } from '...'` no need to declare `const smileys` on line 21.

Reply:
I wasn't seeing the value in this at first, but now I love it! Nice and organized. I can't stop admiring it. Some good looking code!